configure.ac

```diff
@@ -46,6 +46,9 @@ AC_PROG_LIBTOOL
 SPL_AC_LICENSE
 SPL_AC_PACKAGE
 SPL_AC_CONFIG
+SPL_AC_DEBUG
+SPL_AC_DEBUG_KMEM
+SPL_AC_DEBUG_KMEM_TRACKING
```

Comment:
Can you explain why we we move these lines from spl-build.m4 to here?  I don't know what they do.

Reply:
Same as above.